src/Raven.Server/Config/Categories/MonitoringConfiguration.cs

```diff
@@ -30,18 +32,114 @@ public sealed class MonitoringConfiguration : ConfigurationCategory
         public MonitoringConfiguration()
         {
             Snmp = new SnmpConfiguration();
+            OpenTelemetry = new OpenTelemetryConfiguration();
         }
 
         public SnmpConfiguration Snmp { get; }
+        public OpenTelemetryConfiguration OpenTelemetry { get; }
 
         public override void Initialize(IConfigurationRoot settings, HashSet<string> settingsNames, IConfigurationRoot serverWideSettings, HashSet<string> serverWideSettingsNames, ResourceType type, string resourceName)
         {
             base.Initialize(settings, settingsNames, serverWideSettings, serverWideSettingsNames, type, resourceName);
             Snmp.Initialize(settings, settingsNames, serverWideSettings, serverWideSettingsNames, type, resourceName);
-
+            OpenTelemetry.Initialize(settings, settingsNames, serverWideSettings, serverWideSettingsNames, type, resourceName);
             Initialized = true;
         }
 
+        [ConfigurationCategory(ConfigurationCategoryType.Monitoring)]
+        public sealed class OpenTelemetryConfiguration : ConfigurationCategory
+        {
+            [Description("Indicates if OpenTelemetry is enabled or not. Default: false")]
+            [DefaultValue(true)]
+            [ConfigurationEntry("Monitoring.OpenTelemetry.Enabled", ConfigurationEntryScope.ServerWideOnly)]
+            public bool Enabled { get; set; }
+            
+            [Description("Indicates if server-wide OpenTelemetry is enabled or not. Default: true")]
+            [DefaultValue(true)]
+            [ConfigurationEntry("Monitoring.OpenTelemetry.ServerWide.Enabled", ConfigurationEntryScope.ServerWideOnly)]
+            public bool ServerWideEnabled { get; set; }
+            
+            [Description("Indicates if AspNetCoreInstrumentation metrics are enabled or not. Default: true")]
+            [DefaultValue(true)]
+            [ConfigurationEntry("Monitoring.OpenTelemetry.Metrics.AspNetCoreInstrumentation.Enabled", ConfigurationEntryScope.ServerWideOnly)]
+            public bool AspNetCoreInstrumentationMetricsEnabled { get; set; }
+            
+            [Description("Indicates if RuntimeInstrumentation metrics are enabled or not. Default: true")]
+            [DefaultValue(true)]
+            [ConfigurationEntry("Monitoring.OpenTelemetry.Metrics.RuntimeInstrumentation.Enabled", ConfigurationEntryScope.ServerWideOnly)]
+            public bool RuntimeInstrumentationMetricsEnabled { get; set; }
+            
+            [Description("Indicates if metrics should be exported with the OpenTelemetry protocol.")]
+            [DefaultValue(true)]
+            [ConfigurationEntry("Monitoring.OpenTelemetry.OpenTelemetryProtocol.Enabled", ConfigurationEntryScope.ServerWideOnly)]
```

Comment:
Why having "OpenTelemetry" repeated. 
```suggestion
            [ConfigurationEntry("Monitoring.OpenTelemetry.Protocol.Enabled", ConfigurationEntryScope.ServerWideOnly)]
```

Reply:
because `OpenTelemetryProtocol` is official name of their protocol: https://opentelemetry.io/docs/specs/otlp/